src/plugins/utils/js-error-utils.ts

```diff
@@ -60,7 +63,9 @@ const appendErrorObjectDetails = (
     // error may extend Error here, but it is not guaranteed (i.e., it could
     // be any object)
     if (error.name) {
-        rumEvent.type = error.name;
+        rumEvent.type === 'unhandledrejection'
+            ? (rumEvent.type += ': ' + error.name)
+            : (rumEvent.type = error.name);
```

Comment:
issue: Same here. We want to only display the error name if it is available, because this is the current behavior.

Reply:
Agreed, will reflect on next revision.